users/spidey3/rgblight.c

```diff
@@ -0,0 +1,90 @@
+#include QMK_KEYBOARD_H
```

Comment:
For sanity's sake, I would prefer that this was named differently.  It could cause issues with the rgblight feature.

Reply:
Renamed.